dockerfiles/tasks.py

```diff
@@ -39,16 +38,15 @@ def down(c, volumes=False):
     'search': 'Start search container (default: True)',
     'init': 'Perform initialization steps (default: False)',
     'reload': 'Enable automatic process reloading (default: True)',
-    'webpack': 'Start webpack development server (default: False)',
-    'ext-theme': 'Enable new theme from ext-theme (default: False)',
+    'legacy-dashboard': 'Use the old/legacy dashboard (default: False)',
     'scale-build': 'Add additional build instances (default: 1)',
     'http-domain': 'Configure a production domain for HTTP traffic. Subdomains included, '
                    'example.dev implies *.examples.dev for proxito. Example: '
                    '"17b5-139-47-118-243.ngrok.io"',
     'log-level': 'Logging level for the Django application (default: INFO)',
     'django-debug': 'Sets the DEBUG Django setting (default: True)',
 })
-def up(c, search=True, init=False, reload=True, webpack=False, ext_theme=False, scale_build=1, http_domain="", django_debug=True, log_level='INFO'):
```

Comment:
I think this will break with people's existing local dev workflows, so probably also worth a note in Slack to the team about this.